src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$acsentColor: #00acdc;
+$basicWhite: #fff;
+$secondaryColor: #616070;
+$mainAcsent: #060b35;
```

Comment:
There is a typo in the variable name `$mainAcsent`. It should be `$mainAccent` to correctly reflect the term 'accent'.